views/partials/feedback.hbs

```diff
@@ -0,0 +1,27 @@
+{{#if showFeedback}}
+  {{#if debugging}}
+  <script>document.cookie = 'smcx_0_last_shown_at=;expires=Thu, 01 Jan 1970 00:00:01 GMT;';</script>
+  {{/if}}
+  <style>
+/* By default min height is 300 and position
+     in the center (and can't be tweaked).
+     Also we want only screens with enough
+     space to show the pop up.
+
+     TODO: Might be interesting to animate the
+     entrance instead of just popping up
+  */
+.smcx-widget.smcx-modal {
```

Comment:
I think these `!important` rules are unnecessary

Reply:
I believe I had to add it because the CSS from SM was really specific and was added later too so it had higher priority (and there were a few `!important` as well)